mila/dev/sanity_check_runs.md

```diff
@@ -0,0 +1,101 @@
+# Dev Sanity Check Runs
+
+Here is a list of commands to train GFlowNets on various on environments with different configurations that can be used as sanity checks during development of the repository. A number of such runs can be found in [alexhg's `gfn_sanity_checks` wandb project](https://wandb.ai/alexhg/gfn_sanity_checks).
```

Comment:
Good idea to add this. 
Would it be pertinent to add some info on validating those sanity checks? Like "you expect [some metric] to be around [some value]"

Reply:
Actually I expect all the training curves to be relative the same as in previous runs. Shall I write that?

Reply:
How about [this](https://github.com/alexhernandezgarcia/gflownet/pull/178/commits/d4878c1ef5ca250c90c0caa5d9dafa318feb9c11)?

Reply:
Yup, that looks great.